spec/lib/identity_tijuana_push_spec.rb

```diff
@@ -35,7 +35,7 @@
     before(:each) do
       expect_any_instance_of(IdentityTijuana::API).to receive(:tag_emails).with(anything, anything) { {} }
 
-      @external_system_params = JSON.generate({ 'tag' => 'test_tag' })
+      @external_system_params = JSON.generate({ 'tag' => 'test_tag', listcut: 1 })
```

Comment:
I'd suggest using `true` instead of `1` here, using 1 as a boolean is something that's only needed in obsolete languages like C.